File: engine/model/src/main/scala/tools/packet/storage/ArrangeStorage.scala
package tools.packet.storage

import client.inventory.Item
import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ArrangeStorage(private var _slots: Byte, private var _items: java.util.Collection[Item]) extends PacketInput {
  def slots: Byte = _slots

  def items: java.util.Collection[Item] = _items

  override def opcode(): SendOpcode = SendOpcode.STORAGE
}

File: engine/engine-model/src/main/java/tools/packet/storage/ArrangeStorage.java
package tools.packet.storage;

import java.util.Collection;

import client.inventory.Item;
import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ArrangeStorage(Byte slots, Collection<Item> items) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.STORAGE;
   }
}